transport/wsn/src/main/java/org/kaazing/gateway/transport/wsn/WsnConnector.java

```diff
@@ -174,8 +174,8 @@ public void addBridgeFilters(IoFilterChain filterChain) {
 
         // (KG-7391) Use ping and pong to detect and close dead connections, if ws inactivity timeout is active
         final ResourceAddress connectAddress = (ResourceAddress) session.removeAttribute(WSN_CONNECT_ADDRESS_KEY);
-        WsCheckAliveFilter.addIfFeatureEnabled(filterChain, WsnAcceptor.CHECK_ALIVE_FILTER,
-                connectAddress.getOption(WsResourceAddress.INACTIVITY_TIMEOUT), logger);
+//        WsCheckAliveFilter.addIfFeatureEnabled(filterChain, WsnAcceptor.CHECK_ALIVE_FILTER,
+//                connectAddress.getOption(WsResourceAddress.INACTIVITY_TIMEOUT), logger);
```

Comment:
This filter is not needed anymore ?? 
If yes, assigning to connectAddress is not required